.github/workflows/verify-bluehawk.yml

```
@@ -0,0 +1,21 @@
+name: Verify Bluehawk Was Run
```

Minor suggestion: running Bluehawk is an implementation detail - the thing we really want to verify is that the generated output files are present, so I think reflecting that in the check name would be more meaningful to show in the PR checks.
```suggestion
name: Verify Generated Files are Updated
```